npm/builder.js

```diff
@@ -481,8 +479,8 @@ Builder.prototype.compile = function() {
 
   log.title('copy resources');
   log.debug('copy asciidoctor.css');
-  // FIXME: Find a more robust implementation to get the Asciidoctor gem install path
-  var asciidoctorGemPath = child_process.execSync("ruby -e \"print (Gem::Specification.find_by_name 'asciidoctor').full_gem_path\"");
+  // REMIND: Asciidoctor is now a Git submodule
+  var asciidoctorGemPath = 'asciidoctor';
```

Comment:
I'm not building against master because the Git submodule points on the 1.5.4 tag but I don't like git submodules.
We will need to figure out how to avoid downloading the zip each time we are building Asciidoctor.js (ie. `build` directory is deleted)